### PR TITLE
cli, ssl config issue with vault in wildfly-core context

### PR DESCRIPTION
**Pull request by jfdenise**
This is a possible fix for https://issues.jboss.org/browse/WFCORE-1462

### CODE REVIEW

**Comment by aloubyansky** [comment]
Looks good to me. Thanks,  Jean-Francois!

**Comment by bstansberry** [comment]
this is ok to test

File: testsuite/standalone/src/test/java/org/jboss/as/test/integration/management/cli/JBossCliXmlConfigTestCase.java [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+        final CliCommandBuilder commandBuilder = CliCommandBuilder.of(jbossDist);
+        commandBuilder.addJavaOptions("-Djboss.cli.config=" + f.toPath());
+
+        commandBuilder.addCliArgument("--command=help");
+        Process cliProc = null;
+        try {
+            cliProc = Launcher.of(commandBuilder)
+                    .setRedirectErrorStream(true)
+                    .launch();
+            cliProc.waitFor(2000, TimeUnit.MILLISECONDS);
```
Comment (bstansberry) [inline testsuite/standalone/src/test/java/org/jboss/as/test/integration/management/cli/JBossCliXmlConfigTestCase.java]: We have a util that we run all testuite timeouts through to make it easier to make them longer in odd environments. So just as a code standard thing this should be:

cliProc.waitFor(TimeoutUtil.adjust(2000), TimeUnit.MILLISECONDS);

org.jboss.as.test.shared.TimeoutUtil

Also, is 2000ms the appropriate value here? I think the test failure that was reported on the CI run is due to that amount being insufficient.

If the goal of the test isn't to check for something happening quickly, then use a real long timeout (like 30 secs). If the test fails but takes a long time to do so, that's ok; that should be a rare situation. OTOH intermittent CI failures due to short timeouts being exceeded are a real pain.

**Comment by jfdenise** [comment]
I have enlarged and adjusted the timeout.

**Comment by bstansberry** [comment]
Don't worry about the PassivationTestCase failure; that's an annoying intermittent problem. 

It looks like it's still failing on windows though, despite the increased timeout.

**Comment by jfdenise** [comment]
I reproduced the issue on a Windows lab machine. Help command dumps enough content that it makes the Process to hang if out and err streams are not read.

**Comment by aloubyansky** [comment]
Then perhaps use 'version' command instead?

**Comment by aloubyansky** [comment]
retest this please!

**Comment by kabir** [comment]
I think it looks good now, but can you please squash the commits into one?

**Comment by bstansberry** [comment]
When you squash, please also change the commit message to start with [WFCORE-1462]. Thanks!

**Comment by jfdenise** [comment]
Ok, commits squashed

**Pull request merged**